tools/roxvet/analyzers/sortslices/analyzer.go

```diff
@@ -0,0 +1,57 @@
+package sortslices
+
+import (
+	"fmt"
+	"go/ast"
+	"go/types"
+
+	"golang.org/x/tools/go/analysis"
+	"golang.org/x/tools/go/analysis/passes/inspect"
+	"golang.org/x/tools/go/ast/inspector"
+	"golang.org/x/tools/go/types/typeutil"
+)
+
+const doc = `check for usages of "sort" funcs which should be replaced with equivalent "slices" funcs`
+
+// slicesFuncs maps the sort functions to the slices equivalent.
+//
+// Note: sort.IsSorted, sort.Sort, and sort.Stable are not considered, as they take in a sort.Interface which
+// may not necessarily be a slice. Perhaps the usage of these functions should be audited and considered for
+// replacement. The functions sort.Slice and sort.SliceStable should also be considered.
+var slicesFuncs = map[string]string{
+	`sort.Float64s`: `slices.Sort`,
+	`sort.Float64sAreSorted`: `slices.IsSorted`,
+	`sort.Ints`: `slices.Sort`,
+	`sort.IntsAreSorted`: `slices.IsSorted`,
+	`sort.Strings`: `slices.Sort`,
+	`sort.StringsAreSorted`: `slices.IsSorted`,
```

Comment:
You probably need to align the values to start on the same text column. (gofmt -s ...)